convert2rhel/unit_tests/subscription_test.py

```diff
@@ -105,6 +105,17 @@ class PromptUserMocked(unit_tests.MockFunction):
         def __call__(self, *args, **kwargs):
             return True
 
+    class PromptUserLoopMocked(unit_tests.MockFunction):
+        def __init__(self):
+            self.called = 0
+
+        def __call__(self, *args, **kwargs):
+            if self.called >= 1:
+                return "test"
+
+            self.called += 1
+            return ""
```

Comment:
This doesn't do anything in the tests right now, and also stops keeping track if it's been called more than once. One way to test that it loops for each prompt is to keep a list of calls per argument. It should be enough to have a list for first argument like `"Organization: "` and make sure it fails on first call, making it easier to make assertions in the code.

Ideally that should be a separate test though just testing that it loops as it becomes very connected with the code. For the other tests we can use a `DumbCallableObject`